File: mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    //Get Size of Grid
    int n;
    do {
        n = get_int("Size: ");
    } while (n < 1);

    //Print Grid
    for (int i = 0; i < n; i++) {
        for (int j = 0; j < n; j++) {
            printf("#");
        }
        printf("\n");
    }
}

File: wk6/pset/sentimental-mario-more/mario.py
# import get_int
# get input using get_int
# make sure the input is between 1 and 8
# print the pyramids
# for i in range(input - 1):
    # print(" " * (input - i - 1), "#" * i + 1)

from cs50 import get_int

size = 0

while size < 1 && size > 8:
    size = get_int("Height: ")

for i in range(input - 1):
    print(" " * (input - i - 1), "#" * i + 1)

